osidb/models.py

```diff
@@ -217,6 +217,7 @@ class FlawSource(models.TextChoices):
     CORELABS = "CORELABS"
     CUSTOMER = "CUSTOMER"
     CVE = "CVE"
+    CVEORG = "CVEORG"
```

Comment:
@RedHatProductSecurity/osidb-devs FYI, if this gets merged, we'll have to get the Jira field Source (customfield_12324746) updated with the new value. See OSIDB-3352 for an example of that.

Reply:
Hmm, or maybe not if it's not added to FlawSource.allowed? I'm confused.

Reply:
I would say that yes. This is going to become and ordinary and probably frequent flaw source so the trackers should then have that source set and then it must be an allowed Jira field value.